kiali.go

```diff
@@ -47,6 +47,8 @@ import (
 	"github.com/kiali/kiali/status"
 	"github.com/kiali/kiali/tracing"
 	"github.com/kiali/kiali/util"
+
+	_ "go.uber.org/automaxprocs"
```

Comment:
actually, as per our coding guidelines, @antoinedeschenes please move this new import inbetween the two sections above.

Reply:
Should be this:

```go
...
	"strings"
	"syscall"
	"github.com/kiali/kiali/business"
	"github.com/kiali/kiali/business/authentication"

	_ "go.uber.org/automaxprocs"

	"github.com/kiali/kiali/business"
	"github.com/kiali/kiali/business/authentication"
...
```

Reply:
done ✅